playbooks/roles/ocp-lso/tasks/main.yaml

```diff
@@ -0,0 +1,245 @@
+---
+# tasks file for playbooks/roles/ocp-lso
+
+# Check the health of Cluster Operators
+- name: Check if cluster operators and nodes are healthy
+  include_role:
+    name: check-cluster-health
+
+# Checking if worker node contains 500 Gb volume attached
+- name: Check if 500GB volume is attached to worker nodes
+  shell: ssh -o StrictHostKeyChecking=no core@worker-0 lsblk | grep 500
```

Comment:
It should check attached volumes for all the workers right?

Reply:
added loop